app/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ set(FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS
 set(FLATC_ARGS "${FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS}")
 build_flatbuffers("${CMAKE_CURRENT_LIST_DIR}/google_services.fbs"
                   ""
-                  "app_generated_includes"
+                  "app_generated_google_services_includes"
```

Comment:
I added a new build_flatbuffers command below for the heartbeat schema rather than combining it with this build_flatbuffers command because the heartbeat flatbuffer schema is only relevant to the desktop implementation but I think this command is necessary regardless of platform.